nav-challenge/src/components/containers/signUpContainer/SignUpContainer.tsx

```diff
@@ -28,21 +25,24 @@ const SignUpContainer: FC = () => {
     dateOfBirth: "",
   };
 
-  const userDispatch = useDispatchApp();
+  const userDispatch = useDispatch();
```

Comment:
We can and should call all assignments `const dispatch = useDispatch();` so that we can presume that in all of our files they are called the same